charts/v2.1/cray-hms-smd/values.yaml

```diff
@@ -122,13 +122,13 @@ cray-service:
       livenessProbe:
         httpGet:
           port: 27779
-          path: /hsm/v1/service/liveness
+          path: /hsm/v2/service/liveness
```

Comment:
If you are modifying the 2.1 chart, you should probably bump the chart version. Though since the 2.1 chart is being superseded by the 3.0 chart, might make sense to revert this.